extensions/devservices/common/src/main/java/io/quarkus/devservices/common/ContainerAddress.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.devservices.common;
+
+public class ContainerAddress {
```

Comment:
Unfortunately, most of the time it won't be enough.
We may need to add "extra" parameters. 

We can do that later.

Reply:
Yes, I'm aware of that. The current state of this class is to address the combined requirements for:

- Redis
- Kafka
- AMQP

Future cases might require additional fields.